.bazelversion

```diff
@@ -0,0 +1 @@
+7.4.1
```

Comment:
bazel 8.0.0 just released today, The WORKSPACE file is disabled by default, so fix the version temporarily